src/prompt/prompt.js

```diff
@@ -176,3 +177,24 @@ export async function promptForHerokuCredentials() {
 
   return herokuCredentials;
 }
+
+export async function repromptForProjectName(repositories) {
+  console.log(chalk.blue('ReEnter a Valid Project Name: '));
+
+  const question = [{
+    name: 'projectName',
+    type: 'input',
+    message: 'Project Name:',
+    validate: (value) => {
```

Comment:
Will this validation block the user from continuing until they enter a valid name? If not, we'll need a `while` loop above.

Reply:
Yes, this validation is built in to the prompt. Which prevents them from going forward until a name is valid. This is similar to the other prompt validations we have.